File: Python/1929.py
import sys, math

f = lambda : sys.stdin.readline().rstrip()

m, n = map(int, f().split())

for i in range(m, n+1):

    if i == 1: continue;
    tf = True;

    for j in range(2, int(math.sqrt(i)+1)):

        if i%j == 0:

            tf = False;
            break;

    if tf:
        print(i)

File: Java/_1929.java
package Algorithm.BaekJoon;

import java.io.*;
import java.util.*;

public class _1929 {

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        StringTokenizer st = new StringTokenizer(br.readLine());
        StringBuilder sb = new StringBuilder();

        int m = Integer.valueOf(st.nextToken());
        int n = Integer.valueOf(st.nextToken());

        int cnt = 0;
        for(int i = m; i <= n; i++){

            if(i == 1) continue;

            int j;
            boolean tf = true;
            for(j = 2; j <= Math.sqrt(i); j++){

                if(i%j == 0){

                    tf = false;
                    break;
                }
            }

            if(tf){

                sb.append(i + "\n");
            }
        }

        System.out.print(sb);
    }
}
